packages/perseus/src/widgets/image.tsx

```diff
@@ -97,37 +97,20 @@ class ImageWidget extends React.Component<Props> {
 
         const backgroundImage = this.props.backgroundImage;
 
+        if (this.props.alt && this.props.caption && this.props.caption == this.props.alt) {
```

Comment:
Do we need to test for the presence of `alt` and `caption` since their [types are NonNullable](https://github.com/Khan/perseus/blob/0de2bd66e017d6fefc32c5570f83d8c16ce9906a/packages/perseus/src/widgets/image.tsx#L46-L47) (and they have [default values set](https://github.com/Khan/perseus/blob/0de2bd66e017d6fefc32c5570f83d8c16ce9906a/packages/perseus/src/widgets/image.tsx#L74-L75))?